src/ProxyManager/ProxyGenerator/AccessInterceptorScopeLocalizer/MethodGenerator/Util/InterceptorGenerator.php

```diff
@@ -75,9 +76,20 @@ public static function createInterceptedMethodBody(
             . "    \$suffixReturnValue = \$this->$suffixInterceptors" . "[$name]->__invoke("
             . "\$this, \$this, $name, $paramsString, \$returnValue, \$returnEarly);\n\n"
             . "    if (\$returnEarly) {\n"
-            . "        return \$suffixReturnValue;\n"
+            . '        ' . self::returnStatement('$suffixReturnValue', $originalMethod)
             . "    }\n"
             . "}\n\n"
-            . "return \$returnValue;";
+            . self::returnStatement('$returnValue', $originalMethod);
```

Comment:
`rtrim(self::returnStatement('$returnValue', $originalMethod))` on this last statement